### DISCUSSION TITLE
Can you publish a v0.5.4 Flathub release of changes up to this point?

### DISCUSSION THREAD

**Discussion by ourigen** [flxzt/rnote#198] [Development]
First of all thank you for all the great work on this project! Development has been crazy fast, and I think that it's quickly growing into a primary note taking app for many

Since the v0.5.3 release, there has been a number of changes that significantly improve quality of life, (text input, clipboard paste, rebranding, etc.) Seeing that the [v0.6 milestone](https://github.com/flxzt/rnote/milestone/1) may be a long way out, could an intermediate release be made to update the app on Flathub? I think many users would appreciate the recent features

**Comment by flxzt** [comment]
Hi, thanks! Maybe I'll do one in between, but I am definitely planning to publish a beta on flathub soon ( the text feature is not yet entirely stable ). I think at this point it is reasonable to aim for a little more stability than previously, while in general keeping the fast paced releasing.
